WordPress/src/androidTest/java/org/wordpress/android/e2e/ReaderTests.java

```diff
@@ -0,0 +1,63 @@
+package org.wordpress.android.e2e;
+
+import android.Manifest.permission;
+
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.rule.GrantPermissionRule;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.wordpress.android.e2e.pages.ReaderPage;
+import org.wordpress.android.support.BaseTest;
+import org.wordpress.android.ui.WPLaunchActivity;
+
+
+public class ReaderTests extends BaseTest {
+    @Rule
+    public ActivityTestRule<WPLaunchActivity> mActivityTestRule = new ActivityTestRule<>(WPLaunchActivity.class);
+
+    @Rule
+    public GrantPermissionRule mRuntimeImageAccessRule = GrantPermissionRule.grant(permission.WRITE_EXTERNAL_STORAGE);
+
+    @Before
+    public void setUp() {
+        logoutIfNecessary();
+        wpLogin();
+        new ReaderPage().go();
+    }
+
+    @After
+    public void tearDown() {
+        new ReaderPage().dismissReaderViewIfNeeded();
+    }
+
+    String mPostATitle = "Sit Elit Adipiscing Elit Dolor Lorem";
+    String mPostBTitle = "Dolor Sit Elit";
```

Comment:
Are these previously published posts that will always appear on the Reader? Asking cos I don't see a post step but I do see the posts when running the test locally.

Reply:
Yes, I'm using posts already present in `FOLLOWING` tab.